src/ffi/index.ts

```diff
@@ -125,18 +121,24 @@ const initialiseFfi = (logLevel: LogLevel): typeof ffi => {
 };
 
 export const getFfiLib = (
-  logLevel: LogLevel = DEFAULT_LOG_LEVEL
+  logLevel: LogLevel = DEFAULT_LOG_LEVEL,
+  logFile: string | undefined = undefined
 ): typeof ffi => {
   if (!ffi) {
-    logger.trace('Initiliasing ffi for the first time');
-    ffi = initialiseFfi(logLevel);
-  } else {
-    logger.trace('Ffi has already been initialised, no need to repeat it');
-    if (logLevel !== ffiLogLevel) {
-      logger.warn(
-        `The pact native core has already been initialised at log level '${ffiLogLevel}'`
+    logger.trace('Initialising ffi for the first time');
+    ffi = initialiseFfi();
+    logger.debug(
+      `Initialising native core at log level '${logLevel}'`,
+      logFile
+    );
+    if (logFile) {
+      logger.debug(
+        `writing log file at level ${FfiLogLevelFilter[logLevel]} to ${logFile}`
       );
-      logger.warn(`The new requested log level '${logLevel}' will be ignored`);
```

Comment:
Are we missing handling the error in this scenario (somebody initialising the FFI more than once with a different log level)?